src/rust/ain-grpc/src/rpc.rs

```diff
@@ -211,6 +223,36 @@ impl EthServiceApi for EthService {
 
         Ok(EthMiningResult { is_mining: mining })
     }
+
+    fn Eth_GetBlockTransactionCountByHash(
+        handler: Arc<Handlers>,
+        input: EthGetBlockTransactionCountByHashInput,
+    ) -> Result<EthGetBlockTransactionCountByHashResult, jsonrpsee_core::Error> {
+        let EthGetBlockTransactionCountByHashInput { block_hash, .. } = input;
```

Comment:
```suggestion
        let EthGetBlockTransactionCountByHashInput { block_hash } = input;
```

Since there is only one field in the struct, the `..` is unnecessary.